package.json

```diff
@@ -8,7 +8,7 @@
     "test:frontend": "jest --colors",
     "build": "webpack -d --colors",
     "start": "node ./src/server/index.js",
-    "lint": "eslint src/"
+    "lint": "eslint **/*.js"
```

Comment:
it would be fine if the report would be colored
[check documentation](https://eslint.org/docs/user-guide/command-line-interface#options)